app/uk/gov/hmrc/fhregistrationfrontend/views/declaration.scala.html

```diff
@@ -21,22 +21,21 @@
 @import uk.gov.hmrc.fhregistrationfrontend.views.html.helpers._
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
 @import uk.gov.hmrc.fhregistrationfrontend.views.summary.SummaryPageParams
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.layout
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.fhregistrationfrontend.views.helpers.FormErrorMessage.formErrorMessage
 
-@this(
-mainTemplate: uk.gov.hmrc.fhregistrationfrontend.views.html.main_template_old
-)
 
+@this(layout: layout, viewHelpers: ViewHelpers)
 @(declarationForm: Form[Declaration], defaultEmail: Option[String], bpr: BusinessRegistrationDetails, params: SummaryPageParams)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
-@pageScripts = {
-  <script type="text/javascript" src='@routes.Assets.at("javascripts/fh.js")'></script>
-}
 
 @linkVal = @{uk.gov.hmrc.fhregistrationfrontend.controllers.routes.SummaryController.summary.url}
 
 @showAlert = @{
   params.hasAmendments.filter(_ == false).map { _ =>
-    Alert(Messages(s"fh.declaration.${params.mode}.hasAmendments.error"), Some(Messages(s"fh.declaration.${params.mode}.hasAmendments.error.linkText")), Some(linkVal))
+    AlertNew(Messages(s"fh.declaration.${params.mode}.hasAmendments.error"), Some(Messages(s"fh.declaration.${params.mode}.hasAmendments.error.linkText")), Some(linkVal))
```

Comment:
In Ginny's PR when we decided to change the main_template to old, the refactor changed it at all places, when I tried to do the same , the refactor of Alert to AlertOld was not done at all places 🤷‍♀️ , So i changed it to New. I can try again and will let you know, may be My intelliG was behaving weird at that time

Reply:
I was not planning on making a new page for any of the actual view pages unless it is a complicated page, e.g. for businessType page, I updated the existing views/business_Type.scala.html file

Reply:
So , i tried to refactor Alert to AlertOld and it's not refactoring it at all places, So I guess we can keep AlertNew , what do you think?

Reply:
I am making New only for those pages  which are being used at more than one place, so as not to affect other pages

Reply:
We can refactor them later in final cleanup

Reply:
My point was we are going to be doing a clean up at the end because of elements used in more than one place. When we come to clean up it would be easier to just remove the things that are referenced old rather than remove some named old and also some that are the current name and then rename the new ones.

Reply:
Views are a pain for not letting you refactor in all places unfortunately. The way I do it is I right click on the view folder do a "find in file" and type the name of the class then I can see where it needs to be changed. It is a pain but it is going to need to be done at some point either now or later when removing the New.

Reply:
change done